src/main/kotlin/com/github/quiltservertools/ledger/config/SearchSpec.kt

```diff
@@ -4,4 +4,5 @@ import com.uchuhimo.konf.ConfigSpec
 
 object SearchSpec : ConfigSpec() {
     val pageSize by required<Int>()
+    val purgePermissionLevel by required<Int>()
```

Comment:
This would fit better in DatabaseSpec or somewhere else I think

Reply:
Probably, yes